### PR TITLE
Fixed header "content-type: application/json" missing in redact client

### PR DESCRIPTION
**Pull request by sharif-malik**
Added header `Content-Type: application/json` in the redact client. 
As all other clients (sms, voice and others) explicit have this header but it was missing in Redact client. 

## Contribution Checklist
* [x] Unit tests!
* [x] Updated [CHANGELOG.md](CHANGELOG.md)
* [x] My name is in [CONTRIBUTORS.md](CONTRIBUTORS.md)

### CODE REVIEW

**Comment by sharif-malik** [comment]
@cr0wst Please review this PR.

**Comment by sharif-malik** [comment]
Hey Steve(@cr0wst ), 
Could you please review this PR?

Thanks, 
Sharif

**Comment by judy2k** [comment]
> cr0wst Please review this PR.

@sharif-malik please don't bug @cr0wst to review your PR. He receives updates whenever a PR is opened, and our Developer Advocates have a number of priorities - maintaining this library is only one of them.

I can understand that it's frustrating when you have to wait for a review, but I know @cr0wst is extremely dilligent at keeping up to date with this GitHub repo.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.2.2] - 2019-04-11
```
Comment (cr0wst) [inline CHANGELOG.md]:
```suggestion
## [Development]
```

Comment (sharif-malik) [reply CHANGELOG.md]: Updated the PR as per the request.
Thanks!

**Pull request merged**